src/java.base/share/classes/sun/security/ssl/ServerHello.java

```diff
@@ -444,18 +448,17 @@ private static KeyExchangeProperties chooseCipherSuite(
                 if ((hcds == null) || (hcds.length == 0)) {
                     continue;
```

Comment:
I may add a debug log here that the cipher suite is legacy.

Reply:
It is logged in the new method.